core/remote/watcher/index.js

```diff
@@ -13,6 +13,7 @@ const remoteChange = require('../change')
 const { handleCommonCozyErrors } = require('../cozy')
 const { inRemoteTrash } = require('../document')
 const squashMoves = require('./squashMoves')
+const normalizePaths = require('./normalizePaths')
```

Comment:
It goes against my intuition to have to normalize paths on the remote watcher, but I trust you

Reply:
I'm doing it to avoid changing the path saved in PouchDB. It might not be necessary but I thought it would be better to have all paths coherent with each other (e.g. an NFD encoded folder name should be NFD encoded in its children's paths).

I've made some changes and I'm only normalizing on macOS now since Windows and Linux see the same path encoded with 2 different normalizations as 2 different paths and can thus actually perform a move.